integration/chirpstack.md

```diff
@@ -26,9 +26,13 @@ If you are developing your own GPS enabled LoRa device please check the followin
 On the Chirpstack-UI, open your application and then click on the *Integrations* tab on top. Then click on HTTP. In the configuration page for the integration fill in the following:
 
 * **Payload marshaler**: Select `JSON`
+* **Headers**: See [Headers](#headers)
 * **Endpoint**: Add `https://integrations.ttnmapper.org/chirp/v3/events`
 
-### Headers (Optional)
+### Headers
+#### Required
+* **TTNMAPPERORG-NETWORK**: Name of your network, for example: `my.network.name` 
```

Comment:
`Try and make this unique, as this value will be used to distinguish your network's coverage and gateways from other ChirpStack networks' coverage and gateways. This header is a temporary workaround until a proper network identifier is added to ChirpStack. See [this Github issue](https://github.com/brocaar/chirpstack-network-server/issues/532).`